UnitsNet/UnitDefinitions/HeatFlux.json

```diff
@@ -0,0 +1,117 @@
+﻿{
+  "Name": "HeatFlux",
+  "BaseUnit": "WattPerSquareMeter",
+  "XmlDoc": "Heat flux is the flow of energy per unit of area per unit of time",
+  "Units": [
+    {
+      "SingularName": "WattPerSquareMeter",
+      "PluralName": "WattsPerSquareMeter",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Prefixes": [ "Nano", "Micro", "Milli", "Centi", "Deci", "Kilo" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "W/m²" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "WattPerSquareInch",
+      "PluralName": "WattPerSquareInch",
+      "FromUnitToBaseFunc": "x*1.5500031e3",
+      "FromBaseToUnitFunc": "x/1.5500031e3",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "W/in²" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "WattPerSquareFoot",
+      "PluralName": "WattPerSquareFoot",
+      "FromUnitToBaseFunc": "x*1.07639e1",
+      "FromBaseToUnitFunc": "x/1.07639e1",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "W/ft²" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "BtuPerSecondSquareInch",
+      "PluralName": "BtuPerSecondSquareInch",
+      "FromUnitToBaseFunc": "x*1.63533984e6",
+      "FromBaseToUnitFunc": "x/1.63533984e6",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "BTU/s·in²" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "BtuPerSecondSquareFoot",
+      "PluralName": "BtuPerSecondSquareFoot",
+      "FromUnitToBaseFunc": "x*1.13565267e4",
+      "FromBaseToUnitFunc": "x/1.13565267e4",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "BTU/s·ft²" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "BtuPerMinuteSquareFoot",
+      "PluralName": "BtuPerMinuteSquareFoot",
+      "FromUnitToBaseFunc": "x*1.89275445e2",
+      "FromBaseToUnitFunc": "x/1.89275445e2",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "BTU/min·ft²" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "BtuPerHourSquareFoot",
+      "PluralName": "BtuPerHourSquareFoot",
+      "FromUnitToBaseFunc": "x*3.15459075",
+      "FromBaseToUnitFunc": "x/3.15459075",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "BTU/hr·ft²" ]
```

Comment:
Now I look at the others, I suspect I should abbreviate hr to h, correct?

Reply:
Yes we have a guideline to use `h`, unless it would go against a widely used convention.

Reply:
Link to guideline: https://github.com/angularsen/UnitsNet/blob/master/CONTRIBUTING.md#unit-abbreviations